lib/core/src/lib/dialogs/dialog/dialog.component.html

```diff
@@ -37,7 +37,7 @@ <h2 class="adf-dialog-title">{{ data.title | translate }}</h2>
 
     <mat-dialog-content *ngIf="data.contentTemplate || data.contentComponent || data.contentText" class="adf-dialog-content">
         <ng-container [ngTemplateOutlet]="data.contentTemplate"></ng-container>
-        <ng-container [ngComponentOutlet]="data.contentComponent"></ng-container>
+        <ng-container *ngComponentOutlet="data.contentComponent; injector: dataInjector"></ng-container>
```

Comment:
Could you maybe use 'context' instead of 'injector'? Then it will be look a bit shorter because there won't be need to create additional Injection Token

Reply:
discussed on slack